### PR TITLE
[patch] fix pipeline + add PR template + update go.mod

### PR DESCRIPTION
**Pull request by WindzCUHK**
as title

### CODE REVIEW

**Comment by WindzCUHK** [comment]
wait for https://github.com/yahoojapan/athenz-authorizer/pull/66 and update go.mod

File: README.md [hunk 1]
```diff
@@ -92,6 +100,22 @@ This project requires contributors to agree to a [Contributor License Agreement
 
 Note that only for contributions to the garm repository on the [GitHub](https://github.com/yahoojapan/garm), the contributors of them shall be deemed to have agreed to the CLA without individual written agreements.
 
+## About releases
+
+- Merging PR to master will increase the version no. and create a new release automatically.
```
Comment (kyfujisa) [inline README.md:105]: Do you need to write that if PR title doesn't contain these prefixes, versioning in PR will fail?
This part is affected.
- https://github.com/yahoojapan/authorization-proxy/blob/master/.circleci/config.yml#L170

Comment (WindzCUHK) [reply README.md:105]: README is for user. I think it is better to add in PR template.
https://help.github.com/en/github/building-a-strong-community/creating-a-pull-request-template-for-your-repository

Comment (kyfujisa) [reply README.md:105]: >  I think it is better to add in PR template.

ok

> README is for user.

"About releases" describes the commit message.
Is this for user?

Comment (WindzCUHK) [reply README.md]: moved to `CONTRIBUTING.md`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+# for testing (Make sure all the tests pass before you commit and push :))
+make test
+```
+
+## Pull Request Process
+
+- Branch from the master branch and, if needed, rebase to the current master branch before submitting your pull request. If it doesn't merge cleanly with master you may be asked to rebase your changes.
+    - Branches should have descriptive names and start with prefixes like `patch/`, `fix/`, `feature/`. Good examples are: `fix/vulnerability-issue` or `feature/issue-templates`.
+- Commits should be as small as possible, while ensuring that each commit is correct independently (i.e., each commit should compile and pass tests).
```
Comment (kyfujisa) [inline CONTRIBUTING.md:26]: I think this is too harsh.

I think it should be met at stage of merging.

Comment (WindzCUHK) [reply CONTRIBUTING.md:26]: I think it is better to keep in mind, just as a best practice.
We can always squash the commits into one before asking for review.🤪

**Comment by kyfujisa** [comment]
LGTM.

I' m waiting for a fix for the athenz-authorizer update.

File: CONTRIBUTING.md (hunk 2 above)
Comment (kyfujisa) [reply CONTRIBUTING.md]: ok

Comment (WindzCUHK) [reply CONTRIBUTING.md]: remove in 3502b8d

**Pull request merged**